application/common/models/Method.php

```diff
@@ -210,9 +210,13 @@ public static function checkForExistingAndResend($userId, $type, $value)
         ])->all();
 
 
-        $checkFunction = 'mb_strtolower';
+        $checkFunction = function($value) {
```

Comment:
As elsewhere, do we want to make it clear that this is for email addresses (both for readability and so that, if we ever are trying this with something other than a phone number / email address, we'll know to look at this code again)?